src/main/java/net/stickycode/plugin/bounds/StickyCurrentVersionMojo.java

```diff
@@ -34,6 +34,13 @@ public class StickyCurrentVersionMojo
 
   @Parameter(required = false)
   private Map<String, String> coordinates;
+  
+  /**
+   * Useful when using the maven cli to look up the current version of a single artifact. Version range is 
+   * supported since the property is not split into a list.
+   */
+  @Parameter(required = false, property = "artifact")
+  private String artifact;
```

Comment:
Adding a parameter for a singular artifact allows you to get around maven mis-interpreting the comma in a version range.

For example, we could change the `artifacts` parameter to:
```
@Parameter(required=false, property="artifacts")
private List<String> artifacts;
```
which would allow us to set it on the command line via `-Dartifacts=org.apache.commons:commons-lang3:[3,4)`. However, maven will mangle the comma in the version range and interpret this a 2 artifacts:
1. `org.apache.commons:commons-lang3:[3`
2. `4)`
which obviously is not what's intended.